src/main/resources/lang/zh_CN.json

```diff
@@ -1,9 +1,9 @@
 {
   "author": "WolfyScript",
-  "version": "5.0.0",
+  "version": "6.0.0",
```

Comment:
Feel free to put in your name and info here. The author and info are meant as a credit for the translators.
The first translator didn't seem to have done that.

Reply:
Ok...But, The previous translator translated most of the content, he didn't leave any information so I think that I shouldn't use all the author and info 
I'll add myself as a reviser to info, is that OK?

Reply:
All right, revision complete.